megamek/i18n/megamek/common/report-messages.properties

```diff
@@ -195,6 +195,7 @@
 2388=Have a nice trip!
 2390=<data> is landing on top of <data>!
 2395=<newline><data> (<data>) breaks through magma crust on a 6, rolls <data>.
```

Comment:
Noticed this line doesn't match it's nag. Is it a 6+ or a 6?

Reply:
It's a 1d6, so there's no +

Reply:
The nag states 6+

Reply:
Ah, I see. Updated.